### PR TITLE
docs: remove "chore" commit type

### PR DESCRIPTION
**Pull request by dundargoc**
A chore commit type is vague and it can almost always be replaced with a
more descriptive commit type.

### CODE REVIEW

File: scripts/lintcommit.lua [hunk 1]
```diff
@@ -181,7 +181,6 @@ function M._test()
     ['refactor: normal message'] = true,
     ['revert: normal message'] = true,
     ['test: normal message'] = true,
-    ['chore: normal message'] = true,
```
Comment (justinmk) [inline scripts/lintcommit.lua:184]: could stay as a `false` case

**Review by justinmk — APPROVED** [review]
let's also add a "dist" type? (for packaging/distribution/etc)

**Comment by dundargoc** [comment]
> let's also add a "dist" type? (for packaging/distribution/etc)

Hmm, isn't that what the `build` type is for? What would be the difference between them?

**Review by justinmk — APPROVED** [review]

**Comment by justinmk** [comment]
> Hmm, isn't that what the `build` type is for? What would be the difference between `build` and `dist`?

build is more about compilation, generators, etc. 
dist is for packaging, installers, distribution.

Can revisit later though. I'm just basing this on previous examples where "chore" was chosen for dist stuff.

**Comment by dundargoc** [comment]
> build is more about compilation, generators, etc.
dist is for packaging, installers, distribution.

Hmm, I see. I'm not against it but I'm also not sure if we'll need it. I have a feeling it'll be a bit of a "dump"-type like `perf`. I don't mind testing it out though :)

**Review by justinmk — APPROVED** [review]

**Review by mhinz — APPROVED** [review]

**Pull request merged**